### PR TITLE
[Bug 2034296] Inherits storage configs from storage.conf if crio config does not set

### PR DESCRIPTION
**Pull request by QiWang19**
Inherits graphroot, runroot, storage_driver from storage.conf if these configs of crio.conf [crio] is "".
Fix: https://bugzilla.redhat.com/show_bug.cgi?id=2034296
Signed-off-by: Qi Wang <qiwan@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
Inherits storage configs from storage.conf if crio config does not set.
```

### CODE REVIEW

**Comment by haircommander** [comment]
do you think it makes sense to do a similar thing for the other fields in the crio.conf?

File: pkg/config/config.go
Comment (TomSweeneyRedHat) [inline pkg/config/config.go:625]: nit
```suggestion
	// inherits graphroot from storage.conf
```

**Comment by TomSweeneyRedHat** [comment]
One nit, otherwise LGTM, assuming happy tests.

**Comment by QiWang19** [comment]
@haircommander I also added the inherits for `runroot` and `storage_driver`. PTAL

**Comment by haircommander** [comment]
/approve

LGTM, @cri-o/cri-o-maintainers PTAL

**Comment by QiWang19** [comment]
@cri-o/cri-o-maintainers PTAL

**Review by saschagrunert — APPROVED** [review]
LGTM

**Comment by saschagrunert** [comment]
@haircommander PTAL

**Pull request merged**
